internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -27,14 +27,21 @@ const (
 	HighPerformance = "high-performance"
 )
 
+// annotations that should be deprecated
 const (
 	annotationCPULoadBalancing = "cpu-load-balancing.crio.io"
 	annotationCPUQuota         = "cpu-quota.crio.io"
 	annotationIRQLoadBalancing = "irq-load-balancing.crio.io"
-	annotationTrue             = "true"
-	schedDomainDir             = "/proc/sys/kernel/sched_domain"
-	irqSmpAffinityProcFile     = "/proc/irq/default_smp_affinity"
-	cgroupMountPoint           = "/sys/fs/cgroup"
+)
+
+const (
+	annotationDisableCPULoadBalancing = "disable-cpu-load-balancing.crio.io"
+	annotationDisableCPUQuota         = "disable-cpu-quota.crio.io"
+	annotationDisableIRQLoadBalancing = "disable-irq-load-balancing.crio.io"
```

Comment:
Why not just change the misleading "true" value to "disable"? It would make the code more readable I think.

Reply:
done